spec/project_spec.rb

```diff
@@ -701,6 +716,21 @@ class InvalidErrorHandler; end
       expect(spy_logger).to have_received(:log).once.with(Logger::INFO, "Dispatching impression event to URL https://logx.optimizely.com/v1/events with params #{expected_params}.")
       expect(spy_logger).to have_received(:log).once.with(Logger::INFO, "Feature 'multi_variate_feature' is enabled for user 'test_user'.")
     end
+
+    it 'should return false, if the featureEnabled property is false' do
+      allow(project_instance.event_dispatcher).to receive(:dispatch_event).with(instance_of(Optimizely::Event))
+      experiment_to_return = config_body['experiments'][3]
+      variation_to_return = experiment_to_return['variations'][1]
+      decision_to_return = Optimizely::DecisionService::Decision.new(
+        experiment_to_return,
+        variation_to_return,
+        Optimizely::DecisionService::DECISION_SOURCE_EXPERIMENT
+      )
```

Comment:
assert that the given variation's property is indeed false.

Reply:
DONE